frontend/src/scenes/onboarding/OnboardingProductIntro.tsx

```diff
@@ -31,6 +32,18 @@ export const OnboardingProductIntro = ({ product }: { product: BillingProductV2T
             <div className="flex flex-col w-full p-6 bg-mid items-center justify-center">
                 <div className="max-w-lg flex flex-wrap my-8 items-center">
                     <div className="w-1/2 pr-6 min-w-80">
+                        <div className="flex flex-row mb-6">
+                            <LemonButton
+                                to={urls.products()}
+                                icon={<IconArrowLeft />}
+                                type="tertiary"
```

Comment:
tertiary is the default so also rarely needed

Reply:
I hate that tertiary is the default, it makes no sense 😅